src/interbtc-api.ts

```diff
@@ -203,4 +204,8 @@ export class DefaultInterBtcApi implements InterBtcApi {
         // beware: this call will throw if the wrapped currency is not a token!
         return tokenSymbolToCurrency(currencyId.asToken);
     }
+
+    public disconnect(): void {
+        this.api.disconnect();
```

Comment:
I'd suggest adding this as `Promise<void>`, given `ApiPromise.disconnect()` is async.

Reply:
That what my brain wanted to do but somehow the function turned out this way haha